### DISCUSSION TITLE
Use case for double forking a repository.

### DISCUSSION THREAD

**Discussion by johyphenel** [community/community#62623] [Repositories]
### Select Topic Area

Product Feedback

### Body

I have a use case right now where I'd like to fork the same repository twice (with a different name in each case). Since this is not possible today I thought I'd share my feedback, since I think this is a reasonable use case for double forking.

Assuming repo repo_a.

repo_a - fork1: I have some changes and bug fixes I want to make and send pull requests to the original repo, normal github stuff. 

repo_a_gpu - fork2: `repo_a` uses `dep_1` that does workloads on CPU. Someone has made `dep_1_gpu` for better resource efficiency. I need to fork `repo_a`, change all the deps and fix some minor API changes for compatibility. I'd like my own fork here for two reasons, first I want to be able to accept changes from `repo_a`, second I'd like to be able to share my `repo_a_gpu` fork with the modifications for others to use. 

Basically I'd like to make changes and fixes to the core framework in `repo_a` (fork1)  to be able to push to `repo_a` (origin) — but I still want my own copy for `repo_a_gpu` so I can commit and share those changes separate from repo_a, and still get updates from `origin`. 

Edit: Having separate repo is better than having a branch, since it's easier to use separate conda / pip venvs, each with a separate local repo installed for repo_a packages — and it's also easier to direct people to use the right version of repo_a_gpu when it's a clean separate repo, instead of a branch.
